railties/test/application/current_attributes_integration_test.rb

```diff
@@ -0,0 +1,88 @@
+require "isolation/abstract_unit"
+require "rack/test"
+
+class CurrentAttributesIntegrationTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  include Rack::Test::Methods
+
+  setup do
+    build_app
+
+    app_file "app/services/current.rb", <<-RUBY
```

Comment:
For consistency
```diff
- app_file "app/services/current.rb", <<-RUBY
+ app_file "app/models/current.rb", <<-RUBY
```

https://github.com/rails/rails/pull/29180/files#diff-3c3c0f647bc4702f9453c173a707aa06R10